index.html

```diff
@@ -1 +1,152 @@
-hello world
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Epilogue:wght@200;400;500;600;700&family=IBM+Plex+Sans:ital,wght@0,100;0,200;0,300;0,400;0,700;1,200;1,400&display=swap" rel="stylesheet">
```

Comment:
1. As you may have discovered via the course web site, I do love a monospace font and **IBM Plex** is a gorgeous one 😻 
1. In your font stylesheet here, you call a lot of different font weights, but in the CSS you only use the 400 weight in your stylesheet and don't use the **Epilogue** font at all. In general, you want to make sure any unused styles -- including fonts and font weights -- are cleared out before submitting your project.